### PR TITLE
Add remove-component handling on setup_app

### PR DESCRIPTION
**Pull request by commento**
The handling has been added inside the DeviceSelection Menu

### CODE REVIEW

File: senic_hub/setup_app/components/DeviceSelection.js [hunk 1]
```diff
@@ -127,4 +136,25 @@ export default class DeviceSelection extends Screen {
         }
       })
   }
+
+  remove(){
+    console.log('remove')
```
Comment (rudolfs) [inline senic_hub/setup_app/components/DeviceSelection.js]: remove debug logging

File: senic_hub/setup_app/components/DeviceSelection.js [hunk 2]
```diff
@@ -127,4 +136,25 @@ export default class DeviceSelection extends Screen {
...
+    let body = JSON.stringify(component)
+    let params = {
+      method: 'DELETE',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: body,
+    }
+    url = Settings.HUB_API_URL + 'nuimos/' + this.props.nuimoId + '/components/' + this.state.component.id
+    console.log(url)
```
Comment (rudolfs) [inline senic_hub/setup_app/components/DeviceSelection.js]: ditto

File: senic_hub/setup_app/components/DeviceSelection.js (hunk 1 above)
Comment (commento) [inline senic_hub/setup_app/components/DeviceSelection.js]: f1805965b770a68fe124c83280d21c1777d4892b

File: senic_hub/setup_app/components/DeviceSelection.js (hunk 2 above)
Comment (commento) [inline senic_hub/setup_app/components/DeviceSelection.js]: f1805965b770a68fe124c83280d21c1777d4892b

File: senic_hub/setup_app/components/DeviceSelection.js [hunk 3]
```diff
@@ -127,4 +136,23 @@ export default class DeviceSelection extends Screen {
...
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: body,
+    }
+    url = Settings.HUB_API_URL + 'nuimos/' + this.props.nuimoId + '/components/' + this.state.component.id
+    return fetch(url, params)
+      .then(response => {
+        if (!response.ok) {
+          throw new Error('Saving component failed with status: ' + response.status)
```
Comment (luminosuslight) [inline senic_hub/setup_app/components/DeviceSelection.js]: This should probably be `Deleting component [...]`

Comment (commento) [inline senic_hub/setup_app/components/DeviceSelection.js]: yes that's true. thanks, I wil change it in the next PR (I am adding Add feature)

**Pull request merged**